WooCommerce/Classes/System/WooConstants.swift

```diff
@@ -366,6 +366,8 @@ extension WooConstants {
 
         case ordersScreen = "https://woocommerce.com/mobile/orders"
 
+        case customFieldsLearnMore = "https://woocommerce.com/document/custom-product-fields/"
```

Comment:
@hafizrahman I didn't review the PR yet, but I just noticed that we always use the product documentation here, as agreed before (p1727686013865629/1727685068.374159-slack-C03L1NF1EA3), the URL should depend on the `MetaDataType`, and we should use the page https://woocommerce.com/document/managing-orders/view-edit-or-add-an-order/#custom-fields for orders.

Reply:
Good catch. Added in https://github.com/woocommerce/woocommerce-ios/pull/14260/commits/5e29255a927932237a2e71dba8d366ceb63f5f42

Please note that unlike in Android, an existing behavior for the View Custom Fields button in Order Details is that it's hidden by default if the Order has no custom fields:

https://github.com/woocommerce/woocommerce-ios/blob/5a0ddfec620eb33a9c2dafc363e05c88149b8b45/WooCommerce/Classes/ViewModels/Order%20Details/OrderDetailsDataSource.swift#L1235-L1238

So testing the empty state in Order will have to be done by deleting existing custom fields (but without saving the deletion).

Reply:
> Please note that unlike in Android, an existing behavior for the View Custom Fields button in Order Details is that it's hidden by default if the Order has no custom fields:

This behavior was in Android too but I removed it, I think we should do this for iOS too, do you prefer to handle it on this PR or on a separate one?

Reply:
I'll add it here.

Reply:
https://github.com/woocommerce/woocommerce-ios/pull/14260/commits/35f3969ff2a3e6c344d85499d561099446503ea3